### ISSUE TITLE
Docs missing for node platform constraints

### ISSUE THREAD

**Issue by estesp** [docker/cli#619]
As of the latest published docs (17.09), the node constraints for platform that were added in docker/swarmkit#1678 are not documented in the list of constraints shown on the [`docker service create` reference documentation](https://docs.docker.com/engine/reference/commandline/service_create/#specify-service-constraints-constraint)

I can attempt to add a PR for this, but wanted to at least get an issue open for now in case anyone else has more time to attack this.

**Comment by thaJeztah** [comment]
I assume this affects both the CLI reference documentation, and the API docs, correct?

**Comment by estesp** [comment]
that's true; it would affect the API for service create and update as well.
